### DISCUSSION TITLE
Distances to visualise dendrogram for agglomerative clustering

### DISCUSSION THREAD

**Discussion by SJBiomech** [GAA-UAM/scikit-fda#589] [Q&A]
Hi,

I'm looking to use the agglomerative clustering as I think this will be useful for what I want to achieve.

I have looked into visualising the results using a dendrogram but notice that this requires the distances. It seems that I cannot get the distances using this functional agglomerative clustering - is this correct or is there a way to acquire these distances such that I can plot a dendrogram? 

I notice that the original sklearn.cluster.AgglomerativeClustering function has a 'compute_distances' parameter which it states "can be used to make dendrogram visualisation", and also a 'distances_' attribute to acquire them. Is this something that could fairly simply be implemented or achieved with your functional version? (I have tried but been unsuccessful as it is a bit out of my depth unfortunately).

Many thanks!

**Comment by vnmabus** [comment]
The agglomerative clustering class in scikit-fda wraps the one in scikit-learn, although it does not currently expose all of its available methods and attributes. We should review the implementation to expose them, and this should not be a very difficult task.

I have opened #590 to tackle it. PRs are welcome.
